content/docs/working_at_the_turing/reasonable_adjustments.md

```diff
@@ -0,0 +1,26 @@
+---
+# Page title as it appears in the navigation menu
+title: "Reasonable Adjustments"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 4
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Reasonable Adjustments Policy
+
+'Reasonable adjustments' refer to changes to workplaces procedures or practices that are put in place to avoid a situation where a person with disabilities is disadvantaged compared to one without.
+
+The Alan Turing Institute is committed to creating an inclusive, barrier-free workplace, recognising that providing reasonable adjustments is essential to supporting colleagues in fully participating in their roles.
+A summary of the Institute's reasonable adjustment policy can be found [here](https://www.turing.ac.uk/reasonable-adjustments-policy),
+with the full policy available for internal staff on [Mathison](https://thealanturininstitute.sharepoint.com/sites/gc/Public%20Documents/Corporate%20Governance/Policies/People%20Policies/Reasonable%20Adjustment%20Policy%20Final.pdf).
+
+The policy outlines the Institute’s legal obligations and emphasises a proactive approach to making reasonable adjustments, considering each request carefully even if it falls outside of the legal requirements as outlined in the Equality Act 2010.
```

Comment:
Neither the policy summary on the website nor the PDF say that the Turing will consider requests that fall outside our legal requirements. Are we sure this is true? If it is, wouldn't it be better for this statement to be in the policy rather than our handbook?

Reply:
I think point 1.4 does say this:

>  The Institute will go beyond its legal obligations whenever reasonable to do so and our ongoing objective is for an inclusive and diverse workplace. If you have a request for reasonable adjustments, but you are not sure if you strictly fall within the legislation, please do still raise this through the appropriate channels (see Section 9 - relevant contact details) and we will consider all requests very carefully and implement what we reasonably can